sdk/storage/azure-storage-queue/azure/storage/queue/_queue_client.py

```diff
@@ -793,14 +797,15 @@ def update_message(
                 **kwargs)
             new_message = QueueMessage(content=message_text)
             new_message.id = message_id
-            new_message.inserted_on = inserted_on
-            new_message.expires_on = expires_on
-            new_message.dequeue_count = dequeue_count
+            new_message.inserted_on = cast(datetime, inserted_on)
+            new_message.expires_on = cast(datetime, expires_on)
+            new_message.dequeue_count = cast(int, dequeue_count)
             new_message.pop_receipt = response['popreceipt']
             new_message.next_visible_on = response['time_next_visible']
             return new_message
         except HttpResponseError as error:
             process_storage_error(error)
+        return None
```

Comment:
Example of Mypy complaining no return type (since we have no default return) -- how do we want to approach resolving this (new error is return type doesn't match 😄)